facilities/serializers.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework import serializers
+from .models import Owner, Facility, Service
+from common.serializers import AbstractFieldsMixin
+
+
+class OwnerSerializer(AbstractFieldsMixin, serializers.ModelSerializer):
+    class Meta:
+        model = Owner
+
+
+class ServiceSerializer(AbstractFieldsMixin, serializers.ModelSerializer):
+    class Meta:
+        model = Service
+
+
+class FacilityReadSerializer(
+        AbstractFieldsMixin, serializers.ModelSerializer):
+
+    class Meta:
+        model = Facility
+
+
+class FacilitySerializer(
+        AbstractFieldsMixin, serializers.ModelSerializer):
+    services = ServiceSerializer(many=True, required=False)
+
+    def create(self, validated_data):
```

Comment:
You mis-understood my intention in the previous commit; remove this `create` method; the front-enders will use APIs on the join tables directly.